src/index.js

```diff
@@ -31,9 +31,19 @@ let clear
 if (
   perf &&
   perf.mark &&
+  perf.measure &&
   perf.getEntriesByName &&
   perf.getEntriesByType &&
-  perf.clearMeasures
+  perf.clearMarks &&
+  perf.clearMeasures &&
```

Comment:
Seems prudent to check for every API we might use, and not rely on `measure` existing if `mark` exists, and `clearMarks` existing if `clearMeasures` exists.